.gitignore

```
@@ -45,6 +45,7 @@ test/mailers/.keep
 test/models/.keep
 test/system/.keep
 test/test_helper.rb
+test/models/user_test.rb
```

```suggestion
test/
```
ri má
Cả folder test ko care nữa mà